src/Symfony/Component/Validator/Constraints/Timezone.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+* This file is part of the Symfony package.
+*
+* (c) Fabien Potencier <fabien@symfony.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Mickaël Andrieu <andrieu.travail@gmail.com>
+ *
+ */
+class Timezone extends Constraint
+{
+    public $message = 'The value "{{ value }}" is not a valid timezone';
```

Comment:
Also, how about changing the message to `"{{ value }}" is not a valid timezone`? 

`The value "Foo/Bar" is not a valid timezone` doesn't sound right to me.